pom.xml

```diff
@@ -43,6 +43,9 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
+        <!-- bump version when performing releases -->
+        <geomesa.release.version>1.1.0-rc.6</geomesa.release.version>
```

Comment:
Doh.  Of course.  Too bad, there's no ${lastReleasedVersion}.

Manual is fine by me.  The documentation will at least be 1) consistent and 2) quick to fix.